Jellyfin.Api/Controllers/SyncPlayController.cs

```diff
@@ -221,6 +222,21 @@ public async Task<ActionResult> SyncPlayQueue(
         return NoContent();
     }
 
+    /// <summary>
+    /// Gets the current SyncPlay queue.
+    /// </summary>
+    /// <response code="200">Queue returned.</response>
+    /// <returns>A <see cref="QueryResult{SyncPlayQueueItem}"/> containing the current queue items.</returns>
+    [HttpGet("Queue")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [Authorize(Policy = Policies.SyncPlayIsInGroup)]
+    public async Task<ActionResult<QueryResult<SyncPlayQueueItem>>> SyncPlayGetQueue()
+    {
+        var currentSession = await RequestHelpers.GetSession(_sessionManager, _userManager, HttpContext).ConfigureAwait(false);
+        var queue = _syncPlayManager.GetQueue(currentSession);
+        return Ok(new QueryResult<SyncPlayQueueItem>(queue));
+    }
```

Comment:
I want to try enhance this with a BaseItemDto instead of the ID. So if a client wants to utilise this endpoint they dont have to do extra API queries to get the Item from the Id